.travis.yml

```diff
@@ -1,15 +1,13 @@
 language: python
 
 python:
-  - "3.4"
   - "3.5"
   - "3.6"
 
 matrix:
   include:
   - python: 3.7
     dist: xenial
-    sudo: true
```

Comment:
Refer to https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration.

> In the next phase of the migration, all builds will run on virtual-machine-based infrastructure – regardless of the configuration for `sudo` in the `.travis.yml`.

> Soon we will run all projects on the virtual-machine-based infrastructure, the sudo keyword will be fully deprecated.